ios/Podfile.properties.json

```diff
@@ -1,5 +1,6 @@
 {
   "expo.jsEngine": "hermes",
   "EX_DEV_CLIENT_NETWORK_INSPECTOR": "true",
-  "ios.deploymentTarget": "13.0"
+  "ios.deploymentTarget": "13.0",
+  "newArchEnabled": "false"
```

Comment:
This was added after I was playing around with `prebuild`, I decided to keep this change and to remove others. It doesn't hurt since it reflects what we have in the app config file.